packages/twenty-website/src/content/developers/local-setup.mdx

```diff
@@ -23,6 +21,11 @@ Before you can install and use Twenty, make sure you install the following on yo
 You only have to run `corepack enable` to enable Yarn if you haven't done it yet.
 </ArticleWarning>
 
+<ArticleWarning>
+At Twenty, most of the team work on MacOS, so this configuration is the most tested. Linux and Windows work as well, but are more likely to get into issues.
```

Comment:
Maybe `thoroughly tested` will be a better phrasing?

Reply:
After discussing with @FelixMalfait , this warning was removed to be more OS inclusive